evm/src/main/java/org/hyperledger/besu/evm/AccessListEntry.java

```diff
@@ -17,28 +17,48 @@
 import org.hyperledger.besu.datatypes.Address;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.Bytes32;
 
-@JsonSerialize(using = AccessListEntrySerializer.class)
-@JsonDeserialize(using = AccessListEntryDeserializer.class)
```

Comment:
Is this just code simplification, or how does it help with speedup? Just curious.

Reply:
I don't want to drag in JSON deserializers in the jar, which creates a runtime jackson dependency.  Annotations are discarded if the class isn't found so using build time only annotations preserves the jackson auto-serialization while not adding a large library dependency.